### PR TITLE
feat: Change client to stargate version and test code segmentation

### LINKED ISSUE #30: Create a guide documentation
Create guide documents for JAVA SDK interlock

### PR DESCRIPTION
**Pull request by gyuguen**
The JAVA SDK has been modified for the stargate version.
In addition, the test code has been segmentation.

### CODE REVIEW

File: src/main/java/org/medibloc/panacea/PanaceaGrpcClient.java [hunk 1]
```diff
@@ -36,6 +38,15 @@ public BaseAccount getAccount(String address) throws PanaceaApiException {
...
 
+    public Coin getBalance(String address, String denom) {
+        QueryBalanceRequest request = QueryBalanceRequest.newBuilder()
+                .setAddress(address)
+                .setDenom(denom)
+                .build();
+
+        return grpcStub.getBankQueryStub().balance(request).getBalance();
+    }
+
```
Comment (gyuguen) [inline src/main/java/org/medibloc/panacea/PanaceaGrpcClient.java:49]: Unlike before, the quantity of coins does not come out when inquiring by account.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 ## Unreleased
 
-TBD
-
+### Features
+- [\#32](https://github.com/medibloc/panacea-java/pull/32) feat: Upgrade `Stargate` version
```
Comment (youngjoon-lee) [inline CHANGELOG.md]: How about modifying the title to `Make compatible with panacea-core v2.0.0 based on Cosmos SDK Stargate`?

File: src/main/java/org/medibloc/panacea/PanaceaGrpcClient.java (hunk 1 above)
Comment (youngjoon-lee) [reply src/main/java/org/medibloc/panacea/PanaceaGrpcClient.java:49]: Yeah, right.... That's a stupid change from cosmos-sdk...

**Review by youngjoon-lee — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (gyuguen) [reply CHANGELOG.md]: Thank you!
I changed that

**Pull request merged**